pubspec.yaml

```diff
@@ -43,7 +43,10 @@ dependencies:
   flutter_ringtone_player: ^3.1.1
   flutter_secure_storage: ^7.0.1
   flutter_svg: ^0.22.0
-  flutter_typeahead: 4.8.0
+  flutter_typeahead:
+    git:
+      url: https://github.com/linagora/flutter_typeahead.git
+      ref: twake-supported
```

Comment:
why comeback to this library?

Reply:
because it allows me to set context menu of composer to null (disable right click in composer). please merge this one before `TW-1402/cannot-tag-user-in-web`, so i will update it to latest flutter_typeahead